ui/tests/integration/components/pki/pki-issuer-details-test.js

```diff
@@ -0,0 +1,59 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'vault/tests/helpers';
+import { render, settled } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupEngine } from 'ember-engines/test-support';
+import { SELECTORS } from 'vault/tests/helpers/pki/pki-issuer-details';
+
+module('Integration | Component | page/pki-issuer-details', function (hooks) {
+  setupRenderingTest(hooks);
+  setupEngine(hooks, 'pki');
+
+  hooks.beforeEach(async function () {
+    this.context = { owner: this.engine };
+    this.store = this.owner.lookup('service:store');
+    this.secretMountPath = this.owner.lookup('service:secret-mount-path');
+    this.secretMountPath.currentPath = 'pki-test';
+    this.issuer = this.store.createRecord('pki/issuer', { issuerId: 'abcd-efgh' });
+  });
+
+  test('it renders with correct toolbar by default', async function (assert) {
+    await render(hbs`<Page::PkiIssuerDetails @issuer={{this.issuer}} />`, this.context);
+
+    assert.dom(SELECTORS.rotateRoot).doesNotExist();
+    assert.dom(SELECTORS.crossSign).doesNotExist();
+    assert.dom(SELECTORS.signIntermediate).doesNotExist();
+    assert.dom(SELECTORS.download).hasText('Download');
+    assert.dom(SELECTORS.configure).doesNotExist();
+  });
+
+  test('it renders toolbar actions depending on passed capabilities', async function (assert) {
+    this.set('canRotate', true);
+    this.set('canCrossSign', true);
+    this.set('canSignIntermediate', true);
+    this.set('canConfigure', true);
```

Comment:
Oh cool! I didn't know you can do this still! I've been doing something like `this.canRotate = true` in some of other tests

Reply:
Oh, neat! For some reason last time I tried to set directly like that the linter didn't like it. But I will try again, maybe an upgrade fixed it 🎉